contents/tutorials/feedback-interviews-site-apps.md

```diff
@@ -0,0 +1,89 @@
+  posthog.init(
+    'phc_XW7Vk020Cjb1Itg5CMI1Crxm8hdXeTY69qlwmqdoR8m',
+    {
+      api_host:'https://app.posthog.com',
+      opt_in_site_apps: true
+    }
+  )
+</script>
+```
+
+Once added, your app is ready for site apps. PostHog automatically loads site apps into your app through the snippet or library. You don’t need any additional code.
+
+## Setting up the feedback app
+
+The first app we will set up is the feedback widget. To start, search for "Feedback Widget" in your PostHog instance.
+
+![Feedback app](../images/tutorials/feedback-interviews-site-apps/feedback.png)
+
+Click the blue gear to configure domains, styling, content, and data flow related to the app. Customize any of the fields or style how you like (we are keeping the defaults). Click save, then turn on the widget with the toggle next to the hedgehog photo.
+
+Back in your app, you should see an icon in the bottom right corner that enables user feedback. When you (or users) click the icon, a widget pops up with the option to send feedback.
+
+![In app feedback popup](../images/tutorials/feedback-interviews-site-apps/feedback-app.png)
+
+When a user writes feedback and clicks "Send Feedback," PostHog captures it as an event and these events are shown in the "Feedback" tab below Apps in your instance. 
+
+### Integrating the feedback widget into your page
+
+If you want to remove the default icon and integrate it seamlessly into your site, you can set up the feedback widget to trigger on any button or component. To set this up, add `data-attr` to a button or component in your app. For example:
+
+```html
+<button data-attr="posthog-feedback-button">Send feedback</button>
+```
+
+In PostHog, add the name of the `data-attr` to your button configuration. If you want to remove the default icon, also make sure to set "Show feedback button on the page" to "No."
+
+![Config](../images/tutorials/feedback-interviews-site-apps/feedback-config.png)
+
+Once saved, the icon disappears from your app, and clicking the button you set up now triggers the feedback widget.
+
+## Setting up the user interview app
+
+User interviews are critical to understanding how your users are using your product and how you can improve it. The process for booking these interviews can be time-intensive. They require a lot of back-and-forth, figuring out schedules, and coordination. Automating this process helps you to get the benefits of user interviews with less logistic work.
+
+To do this, we will use the user interview app. It is more complicated to set up, because it uses a feature flag. This enables customization and control of who sees the prompt to book a user interview. Here’s the process:
```

Comment:
```suggestion
To do this, we will use [the user interview app](/apps/user-interviews). It is more complicated to set up, because it uses a feature flag, but this enables customization and control of who sees the prompt to book a user interview. Here’s the process:
```